src/pages/kb/user-guide/querying/scheduling-a-query.md

```diff
@@ -26,8 +26,8 @@ We _strongly recommend_ that you configure schedules for any queries tied to [Al
 
 When running queries with parameters on a schedule the scheduler will use the default parameter values for the query execution.
 
+## Failed Scheduled Query Reports
 
-#### Scheduled Queries with Parameters
+If your scheduled query executions fail for some reason, Redash will notify your organizations admin group by email.
```

Comment:
In the original issue I did mention this should be sent out to the organization admin, but eventually I ended up sending these e-mails to the query owner.

Reply:
Fixed this in 491680c136a5b541162164f5ecd5a425fb51acda